devices/chipdb_xilinx.cmake

```diff
@@ -134,8 +137,27 @@ function(generate_xc7_device_db)
         output_name ${device}
```

Comment:
This would need to have a different name than `${device}` or else it will be overwritten (e.g. `${device}_constraints_luts`)

Reply:
Not only that, but also it lead to multiple rules for single device, breaking cmake